src/olympia/editors/models.py

```diff
@@ -757,13 +757,6 @@ class AutoApprovalSummary(ModelBase):
     """Model holding the results of an auto-approval attempt on a Version."""
     version = models.OneToOneField(
         Version, on_delete=models.CASCADE, primary_key=True)
-    uses_custom_csp = models.BooleanField(default=False)
-    uses_native_messaging = models.BooleanField(default=False)
-    uses_content_script_for_all_urls = models.BooleanField(default=False)
-    average_daily_users = models.PositiveIntegerField(default=0)
-    approved_updates = models.PositiveIntegerField(default=0)
-    has_info_request = models.BooleanField(default=False)
-    is_under_admin_review = models.BooleanField(default=False)
```

Comment:
Do we really want to get rid of them in respect to keeping historic data for a little while? Just asking, I don't have a strong opinion on that.

Reply:
Well, they are already unused now. They were only filled with the previous incarnation of auto-approval.

Reply:
Then, nevermind and go on removing them :)